### PR TITLE
Set JDDO threshold to £10,000 (DO NOT MERGE WITHOUT APPROVAL FROM PO)

### PR DESCRIPTION
**Pull request by justin-r-oliver**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-7815

### Change description

Increase threshold for JDDO from < £1000 to ≤ £10,000

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by karthikkannankainos** [comment]
@justin-r-oliver is there no test for featureBuilder?

File: src/main/features/claim/helpers/featuresBuilder.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const featureTogglesClient: FeatureTogglesClient = new FeatureTogglesClient()
 export class FeaturesBuilder {
   static readonly MEDIATION_PILOT_AMOUNT = 500
   static readonly LA_PILOT_THRESHOLD = 300
-  static readonly JUDGE_PILOT_THRESHOLD = 1000
+  static readonly JUDGE_PILOT_THRESHOLD = 10000
```
Comment (karthikkannankainos) [inline src/main/features/claim/helpers/featuresBuilder.ts:12]: is there any test to figure out if it consider case as JDDO if the claim amount is within 300-10000?

Comment (justin-r-oliver) [reply src/main/features/claim/helpers/featuresBuilder.ts:12]: There are 2 tests performed in src/test/features/claim/helpers/featuresBuilder.ts:

1. it(`should add judge pilot eligible to features if amount <= ${FeaturesBuilder.JUDGE_PILOT_THRESHOLD} and flag is set`, async () => {
2. it(`should not add judge pilot eligible to features if amount > ${FeaturesBuilder.JUDGE_PILOT_THRESHOLD}`, async () => {

Comment (karthikkannankainos) [reply src/main/features/claim/helpers/featuresBuilder.ts:12]: thanks @justin-r-oliver

**Pull request merged**